pkg/eval/unix/umask_test.go

```diff
@@ -0,0 +1,40 @@
+// +build !windows,!plan9,!js
+
+// Note that this unit test assumes a UNIX environment with a POSIX compatible
```

Comment:
Do not put comments before the `package` statement; it is only used for package godoc. Put it after the `package` statement.

Reply:
Okay.